src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -3758,6 +3784,779 @@ namespace Google.Example.Library.V1
+            BookName requiredSingularResourceName,
+            BookNameOneof requiredSingularResourceNameOneof,
+            IEnumerable<int> requiredRepeatedInt32,
+            IEnumerable<long> requiredRepeatedInt64,
+            IEnumerable<float> requiredRepeatedFloat,
+            IEnumerable<double> requiredRepeatedDouble,
+            IEnumerable<bool> requiredRepeatedBool,
+            IEnumerable<TestOptionalRequiredFlatteningParamsRequest.Types.InnerEnum> requiredRepeatedEnum,
+            IEnumerable<string> requiredRepeatedString,
+            IEnumerable<ByteString> requiredRepeatedBytes,
+            IEnumerable<TestOptionalRequiredFlatteningParamsRequest.Types.InnerMessage> requiredRepeatedMessage,
+            IEnumerable<BookName> requiredRepeatedResourceName,
+            IEnumerable<BookNameOneof> requiredRepeatedResourceNameOneof,
+            IDictionary<int, string> requiredMap,
+            int? optionalSingularInt32,
+            long? optionalSingularInt64,
+            float? optionalSingularFloat,
+            double? optionalSingularDouble,
+            bool? optionalSingularBool,
+            TestOptionalRequiredFlatteningParamsRequest.Types.InnerEnum? optionalSingularEnum,
+            string optionalSingularString,
+            ByteString optionalSingularBytes,
+            TestOptionalRequiredFlatteningParamsRequest.Types.InnerMessage optionalSingularMessage,
+            BookName optionalSingularResourceName,
+            BookNameOneof optionalSingularResourceNameOneof,
+            IEnumerable<int> optionalRepeatedInt32,
+            IEnumerable<long> optionalRepeatedInt64,
+            IEnumerable<float> optionalRepeatedFloat,
+            IEnumerable<double> optionalRepeatedDouble,
+            IEnumerable<bool> optionalRepeatedBool,
+            IEnumerable<TestOptionalRequiredFlatteningParamsRequest.Types.InnerEnum> optionalRepeatedEnum,
+            IEnumerable<string> optionalRepeatedString,
+            IEnumerable<ByteString> optionalRepeatedBytes,
+            IEnumerable<TestOptionalRequiredFlatteningParamsRequest.Types.InnerMessage> optionalRepeatedMessage,
+            IEnumerable<BookName> optionalRepeatedResourceName,
+            IEnumerable<BookNameOneof> optionalRepeatedResourceNameOneof,
+            IDictionary<int, string> optionalMap,
+            CallSettings callSettings = null) => TestOptionalRequiredFlatteningParamsAsync(
+                new TestOptionalRequiredFlatteningParamsRequest
+                {
+                    RequiredSingularInt32 = requiredSingularInt32,
+                    RequiredSingularInt64 = requiredSingularInt64,
+                    RequiredSingularFloat = requiredSingularFloat,
+                    RequiredSingularDouble = requiredSingularDouble,
+                    RequiredSingularBool = requiredSingularBool,
+                    RequiredSingularEnum = requiredSingularEnum,
```

Comment:
proto requires that the first defined enum value is 0, so I think keeping it like this is OK.